app/models/school.rb

```diff
@@ -24,11 +24,11 @@ class School < ApplicationRecord
   scope :eligible, -> { open.eligible_establishment_type.in_england }
 
   scope :with_name_like, lambda { |search_key|
-    School.eligible.where("name ILIKE ?", "%#{search_key}%")
+    where("name ILIKE ?", "%#{search_key}%")
```

Comment:
Pretty sure we need to scope all school queries with eligible

Reply:
Very much yes, but this is not a query but a scope definition. I would not expect `School.with_name_like('CoE')` to do any additional, surprising filtering, but I agree it should always be called together with `eligible` scope.

Reply:
Are you confident that this does not introduce a behaviour change? Are you confident that everywhere that calls this scope also calls eligible? I agree with the change, I just want to make sure we're not introducing regressions

Reply:
Those scopes are both only used within `search_by_name_or_urn` which in turn is only used in SchoolSearchController together with `eligible` scope. Naturally there is never 100% confidence (especially after being 3 days on the new project :P), but I am pretty sure it should not introduce behavioural change.

Reply:
FWIW I think the answer is "yes" and this is fine :)